epoch/api/README.md

```diff
@@ -31,6 +31,15 @@ The epoch node exposes the following APIs:
     * It is defined in the rest of this document;
     * It is meant to be exposed on the Internet;
     * Its TCP port is configurable.
+  * Range of external Channels noise endpoints. It consists of as many TCP
```

Comment:
Could you also please clarify https://github.com/aeternity/epoch/blob/master/docs/configuration.md#channels for next release?

Reply:
Clarified [in epoch](https://github.com/aeternity/epoch/pull/1667)